### ISSUE TITLE
Shutting down server

### ISSUE THREAD

**Issue by thegamerx1** [AuthMe/AuthMeReloaded#2017]
### What behaviour is observed:
Server shuts down

### What behaviour is expected:
not to shutdown

### Steps/models to reproduce:
turn on server

### Plugin list:
This can be found by running `/pl`

### Environment description
Paper 1.15.2

### AuthMe build number:
This can be found by running `/authme version`
cant start server so i cant run that

### Error Log:
```[21:29:56] [Server thread/INFO]: [AuthMe] Hooked successfully into Essentials
[21:29:56] [Server thread/INFO]: [AuthMe] Essentials spawn file not found: '/var/opt/minecraft/server/plugins/Essentials/spawn.yml'
[21:29:56] [Thread-26/WARN]: [AuthMe] Could not download GeoLiteAPI database [UnknownHostException]: geolite.maxmind.com
[21:29:57] [Server thread/WARN]: [AuthMe] Aborting initialization of AuthMe: [InjectorReflectionException]: Could not invoke constructor of class 'class fr.xephi.authme.command.CommandHandler'
[21:29:57] [Server thread/WARN]: [AuthMe] THE SERVER IS GOING TO SHUT DOWN AS DEFINED IN THE CONFIGURATION!
[21:29:57] [Server thread/INFO]: [AuthMe] Waiting for 0 tasks to finish
```

### Configuration:
Pastebin/Hastebin/Gist link of your config.yml file (remember to delete any sensitive data)
https://pastebin.com/Au1r3XL6

**Comment by thegamerx1** [comment]
after using a newer version it stops with different error
```[21:46:29] [Server thread/INFO]: [AuthMe] Enabling AuthMe v5.6.0-SNAPSHOT-b2377
[21:46:29] [Server thread/WARN]: [AuthMe] Aborting initialization of AuthMe: [NoClassDefFoundError]: ch/jalu/injector/InjectorBuilder
[21:46:29] [Server thread/WARN]: java.lang.NoClassDefFoundError: ch/jalu/injector/InjectorBuilder
[21:46:29] [Server thread/WARN]: 	at fr.xephi.authme.AuthMe.initialize(AuthMe.java:204)
[21:46:29] [Server thread/WARN]: 	at fr.xephi.authme.AuthMe.onEnable(AuthMe.java:145)
[21:46:29] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[21:46:29] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:337)
[21:46:29] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:435)
[21:46:29] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:470)
[21:46:29] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:384)
[21:46:29] [Server thread/WARN]: 	at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:482)
[21:46:29] [Server thread/WARN]: 	at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:298)
[21:46:29] [Server thread/WARN]: 	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:885)
[21:46:29] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:834)
[21:46:29] [Server thread/WARN]: Caused by: java.lang.ClassNotFoundException: ch.jalu.injector.InjectorBuilder
[21:46:29] [Server thread/WARN]: 	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
[21:46:29] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:171)
[21:46:29] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:100)
[21:46:29] [Server thread/WARN]: 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
[21:46:29] [Server thread/WARN]: 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
[21:46:29] [Server thread/WARN]: 	... 11 more
[21:46:29] [Server thread/WARN]: [AuthMe] THE SERVER IS GOING TO SHUT DOWN AS DEFINED IN THE CONFIGURATION!
[21:46:29] [Server thread/INFO]: [AuthMe] Waiting for 0 tasks to finish
[21:46:29] [Server thread/INFO]: [AuthMe] AuthMe 5.6.0-SNAPSHOT-b2377 disabled!
[21:46:31] [Server thread/INFO]: Done (83.299s)! For help, type "help"
[21:46:31] [Server thread/INFO]: Stopping server```

**Comment by thegamerx1** [comment]
fixed the `AuthMe-5.6.0-SNAPSHOT.jar` other than AuthMe-5.6.0-SNAPSHOT-noshade.jar` not sure why did i pick that one though
